config.toml

```diff
@@ -203,6 +203,13 @@ time_format_blog = "02.01.2006"
 # A list of language codes to look for untranslated content, ordered from left to right.
 language_alternatives = ["en"]
 
+[languages.ru]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
+languageName ="Русский"
+weight = 1
+contentDir = "content/ru"
+
```

Comment:
It looks like there should be a `[languages.ru.params]` as well.

Reply:
yes, I added in the last commit! thanks